RELEASE_NOTES.txt

```diff
@@ -1,6 +1,404 @@
+* Fixed priority for field validations
+* Fixed validation in Firefox
+* Update styles of top menu items
+* Update to RxJS 6
+
+
+Node.js:
+* Fixed Node.js thin client UUID marshalling
+
+
+Ignite .NET:
+* Added Thin Client Partition Awareness
+* Added Thin Client Expiry Policy
+* Added Thin Client Cluster API
+* Added .NET Core 3.x support
+* Added Ignite.NET Dockerfile
+* Added Runnable .NET Core assembly
+* Added ICache.GetLongSize
+* Added IClusterNode.Version
+* Added ICompute.WithExecutor
+* Added IIgnite.GetVersion
+* Added IBinaryObjectBuilder.SetField(name, val, type) overload
+* Added Partition Preload API (PreloadPartition, PreloadPartitionAsync)
+* Fixed JVM thread leak
+* Fixed serialization performance for ISerializable
+
+
+Ignite C++:
+* Added BinaryType methods implementations (GetTypeId, GetFieldId)
+* Fixed compilation ODBC and thin client with OpenSSL 1.1
+* Fixed compilation on ubuntu 18.04
+* Implement atomic part of Cache API for C++ thin client (ReplaceIfEquals, RemoveIfEquals etc.)
+* Improve Best Effort Affinity for C++ thin client
+* Remove strong dependency on Boost 1.58.0
+
+
+REST:
+* Added "caches" param for "top" command
+* Added baseline topology commands to REST API
+* Added memory policy metrics via REST
+* Fixed IGNITE_REST_SECURITY_TOKEN_TIMEOUT parameter is set in deciseconds instead of seconds
+
+
+Visor:
+* Added ability to set nodeId for VisorIdleVerifyDumpTask executed from ./control.sh --host HOST --cache idle_verify
+* Added dynamic turn on/off cache statistics for Visor
+* Added guard for cache restarting in progress for Visor tasks.
+* Fixed rebalance status in Visor stays on 99.99%.
+
+
+Control utility:
+* Added ability to check CRC sums of stored pages
+* Added ability to exclude cache from an output file
+* Added ability to show more information about cache configuration
+* Added ability to use simultaneous cache filtering options with control.sh --cache idle_verify
+* Added command to deal with garbage in partitions which left after cache destroy in shared cache groups
+* Added information about coordinator in control.sh output
+* Added showing information about lost on idle_verify
+* Added support SSL to work with baseline configuration (CommandHandler)
+* Added support of changing baseline auto-adjust parameters via console.sh
+* Added support request an SSL Keystore password and SSL truststore password
+* Display time to baseline auto-adjust event in console.sh
+* Fixed connection to cluster error in control.sh while using --cache
+* Fixed control.sh --baseline remove outputs wrong error message when trying to remove a node from baseline
+* Fixed control.sh not working on a multinode cluster setup
+* Fixed idle_verify utility doesn't show both update counter and hash conflicts
+* Fixed incorrect error message after three tries on an unsuccessful authorization
+* Log all action performed thought control.sh
+
+
+ML:
+* Added ML logger and env variables in Spark ML Parser
```

Comment:
add row "Added various Naive Bayes classifiers" and row "Added Gaussian Mixture Model (GMM) trainer with fixed components"